samples/snippets/detect_intent_stream.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""DialogFlow API Detect Intent Python sample with audio files processed as an audio stream.
+
+Examples:
+  python detect_intent_stream.py -h
+  python detect_intent_stream.py --agent AGENT \
+  --session-id SESSION_ID --audio-file-path resources/hello.wav
+"""
+
+import argparse
+import uuid
+
+from google.cloud.dialogflowcx_v3beta1.services.sessions import SessionsClient
+from google.cloud.dialogflowcx_v3beta1.types import audio_config
+from google.cloud.dialogflowcx_v3beta1.types import session
+
+
+# [START dialogflow_detect_intent_stream]
+def detect_intent_stream(agent, session_id, audio_file_path, language_code):
```

Comment:
Please add some hints to the user to what these values mean. You can do one of either of the following:

(preferred) Add a simple overloaded method that demonstrates initalization and calles the underlying function:
```python 
# [START dialogflow_detect_intent_stream]
def run_sample():
    # TODO(developer): Replace these values when running the function
    agent = "YOUR-AGENT-ID"  # https://link.to.more.info.on.agents
    # ...
    detect_intent_stream(agent, session_id, audio_file_path, language_code):

def detect_intent_stream(agent, session_id, audio_file_path, language_code):
```